### PR TITLE
Show string/word numbers on search results

### PR DESCRIPTION
**Pull request by JagdishAneshwar**
# Summary:
This pull request adds the display of total strings and words in search results. Issue: #11853

# Changes Introduced:
- Updated the search view to calculate total_strings and total_words from the search results.
- Modified search.html to show total_strings and total_words at the top of the search results.

### CODE REVIEW

File: weblate/trans/views/search.py [hunk 1]
```diff
@@ -154,6 +154,11 @@ def search(request, path=None):
         units = get_paginator(
             request, units.order_by_request(search_form.cleaned_data, obj)
         )
+
+        # Calculate total number of strings and words
+        total_strings = units.paginator.count
+        total_words = sum(unit.num_words for unit in units.object_list)
```
Comment (nijel) [inline weblate/trans/views/search.py:160]: The strings number is for all strings, but words number just for the current page.

File: weblate/templates/search.html [hunk 2]
```diff
@@ -19,6 +19,7 @@
         {% crispy search_form  %}
     </form>
     {% if page_obj.object_list %}
+        <div><p>{{ total_strings }} strings / {{ total_words }} words</p></div>
```
Comment (nijel) [inline weblate/templates/search.html:22]: This needs localization.

**Review by JagdishAneshwar — COMMENTED** [review]
I've updated the code, pls check

File: weblate/trans/views/search.py [hunk 3]
```diff
@@ -157,7 +157,7 @@ def search(request, path=None):
 
         # Calculate total number of strings and words
         total_strings = units.paginator.count
-        total_words = sum(unit.num_words for unit in units.object_list)
+        total_words = sum(unit.num_words for unit in unit_set.all())
```
Comment (nijel) [inline weblate/trans/views/search.py:160]: This will be super slow on huge results, this needs a database aggregation.

**Pull request closed without merging**